.circleci/config.yml

```diff
@@ -109,3 +119,28 @@ jobs:
                 -r ${CIRCLE_PROJECT_REPONAME} \
                 -c ${CIRCLE_SHA1} \
                 v${VERSION} ./dist/
+
+  push-image:
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Setup Auth
+          command: |
+            echo ${GCLOUD_SERVICE_KEY} > ${HOME}/gcp-key.json
+            gcloud auth activate-service-account --key-file ${HOME}/gcp-key.json
+            gcloud --quiet config set project ${GOOGLE_PROJECT_ID}
+      - run:
+          name: Build docker image
+          command: docker build --rm=false -t gcr.io/gapic-showcase/gapic-showcase .
```

Comment:
what's the `--rm` option for?

Reply:
Controls removing intermediate containers. I actually dont think it's needed. Removing.